spotinst_sdk2/__init__.py

```diff
@@ -53,6 +53,8 @@ def client(self, service, print_output=True, log_level=None, user_agent=None, ti
                                               user_agent=user_agent, timeout=timeout),
             "ocean_aws": OceanAwsClient(session=self.session, print_output=print_output, log_level=log_level,
                                         user_agent=user_agent, timeout=timeout),
+            "ocean_aws_rightsizing": OceanRightSizingClient(session=self.session, print_output=print_output, log_level=log_level,
```

Comment:
"ocean_aws_rightsizing" should be changed to "ocean_rightsizing"

Reply:
changed